.github/actions/setup/action.yml

```diff
@@ -20,19 +20,20 @@ runs:
       with:
         version: nightly
 
-    # Workaround for parallel building with forge:
-    # https://github.com/foundry-rs/foundry/issues/4736
-    - name: Install solc
-      shell: bash
-      run: cargo install svm-rs && svm install 0.8.13
-
     - name: Setup protoc
       uses: arduino/setup-protoc@v1
 
     - name: Install node dependencies
       shell: bash
       run: pnpm install --frozen-lockfile
 
+    # Workaround for parallel building with forge:
+    # https://github.com/foundry-rs/foundry/issues/4736
+    - name: Setup forge
+      shell: bash
+      working-directory: packages/schema-type
```

Comment:
this felt like the simplest solidity package to build with no dependencies on other workspace things